target/linux/ramips/image/mt76x8.mk

```diff
@@ -1097,6 +1097,15 @@ define Device/xiaomi_miwifi-nano
 endef
 TARGET_DEVICES += xiaomi_miwifi-nano
 
+define Device/xiaomi_miwifi-3a
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := MiWiFi 3A
+  DEVICE_PACKAGES := uboot-envtools kmod-mt76x2
+  SUPPORTED_DEVICES += miwifi-3a
```

Comment:
`SUPPORTED_DEVICES` should not be needed.

You need it hen you want to do sysupgrade from a different OpenWrt fork which named the device `miwifi-3a` and not `xiaomi_miwifi-3a`.